### PR TITLE
Add more stuff in the README

### PR DESCRIPTION
**Pull request by marcofugaro**
Hey, I thought it would be necessary to add in the Readme what improvements were done in this fork.

Please let me know if I missed something or you have better phrasing for anything 🙏

### CODE REVIEW

**Comment by stockHuman** [comment]
Looks good! Do we also wanna point people to use-cannon?

**Comment by marcofugaro** [comment]
@stockHuman great idea!

File: readme.md [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
 
-    yarn add cannon-es
+It's a type-safe flatbundle (esm and cjs) which allows for **tree shaking** and usage in **typescript** environments.
 
-```jsx
+These minor changes and improvements were also made:
+
+- These PRs from the original repo were merged: [schteppe/cannon.js#433](https://github.com/schteppe/cannon.js/pull/433), [schteppe/cannon.js#430](https://github.com/schteppe/cannon.js/pull/430), [schteppe/cannon.js#418](https://github.com/schteppe/cannon.js/pull/418), [schteppe/cannon.js#360](https://github.com/schteppe/cannon.js/pull/360), [schteppe/cannon.js#265](https://github.com/schteppe/cannon.js/pull/265)
+- The `ConvexPolyhedron` constructor now accepts an object instead of a list of arguments [#6](https://github.com/react-spring/cannon-es/pull/6)
+- `World.activeBodies: Body[]` has been replaced with `World.hasActiveBodies: boolean`
```
Comment (codynova) [inline readme.md:9]: @marcofugaro One small correction here, `World.activeBodies` was something I added, so probably not worth mentioning - it was only in `cannon-es` for one version I think.

Comment (marcofugaro) [reply readme.md]: Ups, I just looked at the commits and didn't know the context.

Sure! Could you replace that line with a brief description of the frame invalidation thing? 🙏

Comment (marcofugaro) [reply readme.md]: Seen it now, thank you! 🙂

**Pull request merged**